src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width,
+               user-scalable=no,
+               initial-scale=1.0,
+               maximum-scale=1.0,
+               minimum-scale=1.0"
+    >
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="wrap">
+      <div class="card" data-qa="card">
+        <img class="card__img" src="images/imac.jpeg"
+          alt="imac">
+
+        <h2 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+
+        <p class="card__code">Код товара: 195434</p>
+
+        <div class="card__rating rating">
+          <div class="rating__stars rating__stars--3">
+            <span class="rating__star"></span>
+            <span class="rating__star"></span>
+            <span class="rating__star"></span>
+            <span class="rating__star"></span>
+            <span class="rating__star"></span>
+          </div>
+          <div class="rating__review">Отзывов: 5</div>
+        </div>
+
+        <div class="card__price price">
+          <div class="price__item">Цена:</div>
+          <div class="price__uah">69 999 грн</div>
+        </div>
+
+        <a href="#" class="card__button"
+          data-qa="hover">
+            Купить
+        </a>
```

Comment:
I see more than one mistake like this

```suggestion
        <a
          href="#"
          class="card__button"
          data-qa="hover"
        >
            Купить
        </a>
```